page.md

```diff
@@ -39,3 +41,61 @@ The `comments` attribute of the `<pg:AlternativeImage>` attribute should be used
 For the results of image processing that changes the positions of pixels (e.g. cropping, rotation, dewarping), `AlternativeImage` on page level and polygon of recognized zones is not enough to access the image section a region is based on since coordinates are always relative to the original image.
 
 For such use cases, `<pg:AlternativeImage>` may be used as a child of `<pg:TextRegion>`, `<pg:TextLine>`, `<pg:Word>` or `<pg:Glyph>`.
+
+# Font Information
+
+## Font family and typographic style plus characteristics
+
+The font information (type, cut...) are documented in two places in PAGE XML.
+
+**The first value:**
+
+As custom value for the elements: `<TextLine>`, `<Word>`, it is also possible  `<TextRegion>`
+here an example for *Textline*
+
+```xml
+<TextLine custom="textStyle {fontFamily:Arial; fontSize:17.0; bold:true;}">
+```
+
+The keyword for this information is textStyle. For the font: fontFamily, for the size: fontSize and for the typographic style the characteristic feature. 
+See: http://www.ocr-d.de/sites/all/gt_guidelines/lyTypographie.html
+
+**The second value:**
+
+This information is **primarily recorded in the `<TextStyle>` element**. 
+See: http://www.ocr-d.de/sites/all/gt_guidelines/pagecontent_xsd_Complex_Type_pc_TextStyleType.html?hl=textstyle
+
+All Information are documented in element `<TextStyle>`
+
+```xml
+<TextStyle fontFamily="Arial" fontSize="17.0" bold="true"/>
+```
+
+However, since not all typographic information can be stored in TextStyle, this is in the case of:
+- antiqua
+- fracturswitch
+- doubleunderlined
+to use the custom attribute value.
```

Comment:
What If information in TextStyle is provided and clashes with information in custom? 

Won't data consumers be required to parse `custom` anyway, if it can be redundant?

Can `TextStyle` be used whereever we need it or do we need to change PAGE XML?